README.md

```diff
@@ -62,20 +98,22 @@ $ dog-food -d my-chat.db
 If you deployed your own SMP server you can set client to use it via `-s` option:
 
 ```shell
-$ dog-food -s smp.example.com:5223
+$ dog-food -s smp.example.com:5223#KXNE1m2E1m0lm92WGKet9CL6+lO742Vy5G6nsrkvgs8=
 ```
 
+The server will generate RSA transport key on the first run and output key digest every time you run it.
+
 You can still talk to people using default or any other server, it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
```

Comment:
add to roadmap that in future multiple servers can be configured for queue initialization as well via chat app configuration?

Reply:
not sure, it's just this phrasing implies you can only use 1 server for queues you initialize (which seems true for current version of chat..)